### ISSUE TITLE
app: Vrm model's names should get updated in character tab and diorama.

### ISSUE THREAD

**Issue by amnaarshed4224** [webaverse/app#2411]
Right now the character tab and diorama both, give the same name for all models "Scillia". Names should get updated based on the model being equipped.

![vrm111](https://user-images.githubusercontent.com/35926530/154952931-af89f4e0-a5bc-439d-8136-f01744429472.PNG)
![vrm222](https://user-images.githubusercontent.com/35926530/154952938-2ceeb658-5392-4075-9282-b9fd7fc3fb55.PNG)

**Comment by avaer** [comment]
Yes, we need to finish this part of the UI pulling from the login.
